platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/MerkleDb.java

```diff
@@ -574,7 +650,14 @@ private void storeMetadata(final Path dir, final Collection<TableMetadata> table
     }
 
     private AtomicReferenceArray<TableMetadata> loadMetadata() {
-        return loadMetadata(storageDir);
+        final AtomicReferenceArray<TableMetadata> metadata = loadMetadata(storageDir);
+        // All tables loaded from disk are primary
```

Comment:
Will this have any issues with duplication the first time it loads state from disk?  That is, is there any chance the pre-existing code stored both primary and secondary tables?